include/kframework/lemmas/bytes-simplification.k

```diff
@@ -68,50 +68,155 @@ module BYTES-SIMPLIFICATION [symbolic]
 
     // #range
 
-    rule #range(_, _, K) => .Bytes requires K <=Int 0 [simplification]
+    rule [range-empty]:
+      #range(_:Bytes, S:Int, W:Int) => .Bytes
+      requires S <Int 0 orBool W <=Int 0
+      [simplification(1)]
```

Comment:
```suggestion
      [simplification(40)]
```

Any reason to choose 1 in particular? That leaves little room for inserting rules before it.

Reply:
Good point, I'll change to something more flexible.